Mathlib/Combinatorics/SimpleGraph/Hasse.lean

```diff
@@ -121,4 +121,20 @@ theorem pathGraph_two_eq_top : pathGraph 2 = ⊤ := by
   ext u v
   fin_cases u <;> fin_cases v <;> simp [pathGraph, ← Fin.coe_covby_iff, Nat.covby_iff_succ_eq]
 
+/-- Hommorphism from smaller path graph to bigger path graph· -/
+def pathGraph_self_Hom {n m : ℕ} (hnm : n ≤ m) : pathGraph n →g pathGraph m where
```

Comment:
Maybe
```suggestion
protected def Hom.pathGraph {n m : ℕ} (hnm : n ≤ m) : pathGraph n →g pathGraph m where
```
is a better name?

Reply:
I'm not sure of how `protected` works in these case, but if I put the def as `protected def Hom.pathGraph` when I try to use it in `ConcreteColorings.lean` as `Hom.pathGraph` I get these error:
```
invalid field notation, type is not of the form (C ...) where C is a constant
  Hom
has type
  SimpleGraph ?m.141359 → SimpleGraph ?m.141360 → Type (max ?u.141358 ?u.141357)
```
I tried several ways of using it but non of them worked

Reply:
There's a slight chance it is broken, but the more likely explanation is that you did not provide the correct number of arguments.

Reply:
Yes, provably it was something like that becase when I tried again it worked